lib/services/stripe.ts

```diff
@@ -31,31 +31,56 @@ const LOCALES = new Map<'fr-FR' | 'en-US', StripeElementLocale>([
 ]);
 
 export class StripeService {
-  instance: Stripe | null = null;
-  elements: StripeElements | null = null;
+  private isInit = false;
+  private config: StripeConfig;
+  private instance: Stripe | null = null;
+  private elements: StripeElements | null = null;
```

Comment:
Seems not necessary, we loose access on key components of stripe here.
Would force to make a specific function for each use case :/ is this the goal?

Reply:
There's no such case in megashark-lib, is there in Parsec?
It is absolutely the goal: the service should always be the one responsible. And it is necessary: the problem we have is that Stripe is accessed without having been initialized first (because the user is offline for example). The functions exposed by the service can re-try to init Stripe before using the components. If we expose the stripe components directly, we can't do that.